src/main/java/org/apache/maven/shared/utils/logging/Style.java

```diff
@@ -31,15 +31,18 @@
 enum Style
 {
 
-    DEBUG(   "bold,cyan"   ),
-    INFO(    "bold,blue"   ),
-    WARNING( "bold,yellow" ),
-    ERROR(   "bold,red"    ),
-    SUCCESS( "bold,green"  ),
-    FAILURE( "bold,red"    ),
-    STRONG(  "bold"        ),
-    MOJO(    "green"       ),
-    PROJECT( "cyan"        );
+    DEBUG(       "bold,cyan"   ),
+    INFO(        "bold,blue"   ),
+    WARNING(     "bold,yellow" ),
+    ERROR(       "bold,red"    ),
+    SUCCESS(     "bold,green"  ),
+    FAILURE(     "bold,red"    ),
+    STRONG(      "bold"        ),
+    MOJO(        "green"       ),
+    PROJECT(     "cyan"        ),
+    WORKING_DIR( "black"       ),
```

Comment:
That is going to be hard if your terminal is black.

Reply:
Fair enough, didn't think of that. Not specifying a style here gives you black letters on a light terminal, and white letters on a black terminal. How about that?

Reply:
We need to experiment with this to see what looks best. I've been thinking about green / blue / bold+blue.
I think we should stick to 2 colors, making 1 bold as well.

Reply:
Sure, we could also restrict ourself to using less colours. I don't have a strong preference, so whatever you think is best suited...